homeassistant/components/todoist/services.yaml

```
@@ -22,6 +22,12 @@ new_task:
       example: Chores,Delivieries
       selector:
         text:
+    assignee:
+      name: Assignee
+      description: A person in a shared project. to assign this task to.
```

Minor: Got an extra `.` that isn't needed.

```suggestion
      description: A person in a shared project to assign this task to.
```

---

One thing that was confusing when I was testing this out is what value to use for the assignee.  We might want to call out that this is the full name of the user, e.g. `Dave Smith`.

---

I was confused as well that the attribute is full_name but it seems not be always the `full name`.How did you create your Todoist account? Cause with our Google SSO  accounts it's not the full name but the part infront of the @. In my case `kai.bepperling`

---

I signed up just using an email address, so you are right there is a difference based on which signup method was used.  The generic approach seems fine to me, it might be worth going into a little more detail in a docs PR for the component.  Have you created one of those too? If not, it would be worthwhile to update the docs there with this new attribute for the service: https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/todoist.markdown

---

I have it on my list to update the documentation as well. Will probably open the PR this weekend.